core/data/clients/InfluxEvent.go

```diff
@@ -1,20 +1,10 @@
-/*******************************************************************************
- * Copyright 2017 Dell Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License. You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software distributed under the License
- * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
- * or implied. See the License for the specific language governing permissions and limitations under
- * the License.
- *
- * @microservice: core-data-go library
- * @author: Ryan Comer, Dell
- * @version: 0.5.0
- *******************************************************************************/
+//
+// Copyright (c) 2018
+// m-masataka
```

Comment:
One more thing - if your code is based on Dell's code (i.e. re-write and not "from scratch" original code) - you should keep Dell as a copyright owner as well.

Reply:
Ok. This code is based on Dell's code.
So, keep Dell's copyright. 
thank you.